### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zhangqianhui**
self-supervised learning with generative adversarial networks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -409,6 +409,16 @@ FAIR Self-Supervision Benchmark [[repo]](https://github.com/facebookresearch/fai
   [[pdf]](https://papers.nips.cc/paper/7557-playing-hard-exploration-games-by-watching-youtube.pdf) 
   - Yusuf Aytar, Tobias Pfaff, David Budden, Tom Le Paine, Ziyu Wang, Nando de Freitas. *NIPS 2018*
 
+### Adversarial Learning
+- Self-Supervised GANs via Auxiliary Rotation Loss. 
+  [[pdf]](https://arxiv.org/pdf/1811.11212.pdf) 
+  [[code]](https://github.com/zhangqianhui/Self-Supervised-GANs)
+  - Deepak Pathak, Pulkit Agrawal, Alexei A. Efros, and Trevor Darrell. *CVPR 2019*
```
Comment (jason718) [inline README.md]: author list is wrong.

File: README.md [hunk 2]
```diff
@@ -409,6 +409,16 @@ FAIR Self-Supervision Benchmark [[repo]](https://github.com/facebookresearch/fai
   [[pdf]](https://papers.nips.cc/paper/7557-playing-hard-exploration-games-by-watching-youtube.pdf) 
   - Yusuf Aytar, Tobias Pfaff, David Budden, Tom Le Paine, Ziyu Wang, Nando de Freitas. *NIPS 2018*
 
+### Adversarial Learning
+- Self-Supervised GANs via Auxiliary Rotation Loss.
```
Comment (jason718) [inline README.md]: I have already added this paper actually.

**Comment by zhangqianhui** [comment]
New commit

**Comment by jason718** [comment]
will merge later once these paper get accepted.

**Comment by zhangqianhui** [comment]
ok

Jason Ren <notifications@github.com> 于2019年6月13日周四 上午6:47写道：

**Pull request closed without merging**